sample/ApplicationInsightsHostingStartupSample/Properties/launchSettings.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
You do not need launch.json if the app only used for integration tests, are you? Please remove to avoid confusion